web/lib/firebase/notifications.ts

```diff
@@ -0,0 +1,27 @@
+import { collection, query, where } from 'firebase/firestore'
+import { Notification } from 'common/notification'
+import { db } from 'web/lib/firebase/init'
+import { getValues, listenForValues } from 'web/lib/firebase/utils'
+
+function getNotificationsCollection(userId: string, unseenOnly?: boolean) {
+  const notifsCollection = collection(db, `/users/${userId}/notifications`)
+  if (unseenOnly) return query(notifsCollection, where('isSeen', '==', false))
+  return query(notifsCollection)
+}
+
+export function getUnseenNotifications(userId: string) {
```

Comment:
Should be pretty easy to listen for unseen notifications, right? (Use listenForValues instead of getValues, then create a hook.) Seems way easier and less hacky than polling every 5 seconds.

Reply:
Is there an extra cost to listening? I figured people don't need them real time so toned down the urgency a bit with a poll. Happy to listen though, already doing so elsewhere

Reply:
Basically no. Firebase listeners are very lightweight/low-cost. There's no reason to do polling instead.

Reply:
thanks for the tips!